cmd/webhook/admission/testdata/ingress/invalid-ingress-with-duplicate-hosts.json

```diff
@@ -0,0 +1,41 @@
+{
+    "request": {
```

Comment:
I still think we do not need to add tests for admission webhook when we change validator, its just increases size of PR but adds no value as webhook simply calls validator